app/views/scripts/show.html.erb

```diff
@@ -0,0 +1,66 @@
+
+<div class="page-header" >
+  <h2><%= @script.name %></h2>  <!-- but also title type header  -->
+</div>
+
+<script>pollForUpdates('<%= @project.id %>', '<%= @script.id %>')</script>
+
+<%= link_to 'Back', project_path(@project), class: "btn btn-info" %><br><br>
+
+<h3>All the job results for <%= @script.name%></h3>
+
+<div id="script-list-view" class="panel panel-default">
```

Comment:
For this page a table is better than the mixture of panel and tables and progressive disclosure. This page shows a list of jobs and a little bit of information about each: job id, job status, created_at, updated_at, output file location

This is similar to the amount of information for the list views of all of our workflow apps in AweSim and OnDemand, and those apps use tables to display them.

Reply:
FWIW if you come across a situation in the future, where you have a list view and there is too much information per row to display that you want to move to progressive disclosure, its better to first add a details view instead - that takes a lot less code and uses boring Rails idioms to accomplish (i.e. no JavaScript, views look like views in other apps and vanilla Rails apps).